lib/angular-evaporate.js

```diff
@@ -113,6 +113,16 @@
             this.contentType = this.file.type || 'binary/octet-stream';
             this.$url = ae.$url + this.name;
 
+            function readableFileSize(size) {
```

Comment:
Oh, you're right, sorry( I probably thought about every `AngularEvaporateUpload` instance